app/controllers/YearsController.scala

```diff
@@ -41,7 +43,8 @@ class YearsController @Inject()(
                                 val controllerComponents: MessagesControllerComponents,
                                 renderer: Renderer,
                                 config: FrontendAppConfig,
-                                schemeService: SchemeService
+                                schemeService: SchemeService,
+                                yearsView: YearsView
                             )(implicit ec: ExecutionContext)
                                 extends FrontendBaseController
                                 with I18nSupport
```

Comment:
sorry forgot to comment on this one
Please remove this with NunjucksSupport and import